### PR TITLE
Shelter types fixes

### PR DESCRIPTION
**Pull request by hunterowens**
unfortunately needed a new layer, but here are the shelter types fixes.

### CODE REVIEW

File: dags/public-health/covid19/README.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ We are using the Johns Hopkins data for our ETL and ESRI feature services. Their
...
 
+## Shelter Data
+
+The DAG `shelter-to-esri.py` takes the Rec + Parks shelter census (collected at 8 intervals a day) and pushes it into the City of LA geohub by merging it with the the official shelter data from LASAN / RAP GIS staff. We do some timezone data cleaning and publish.
+
+`Timestamp` is the time in which the shelter actually submitted the Google form. `Date` and `Time` are which "report" they are filing for. 
+
+Note, the capacity numbers should be calculated by `sum(occupied beds + unoccupied beds)`, rather than the normal capacity, which has been lower to help adhear to social distancing in the shelters.
```
Comment (ian-r-rose) [inline dags/public-health/covid19/README.md]: "adhere"

File: dags/public-health/covid19/shelter-to-esri.py [hunk 2]
```diff
@@ -67,14 +67,20 @@ def load_data(**kwargs):
     
     # timestamp 
     df['Timestamp'] = pd.to_datetime(df.Timestamp).dt.tz_localize(tz="US/Pacific").dt.tz_convert('UTC')
+    df['string-ts'] = df['Timestamp'].dt.strftime('%B %d, %Y, %r')
+
+    df['string-ti'] = df['Time'].dt.strftime('%B %d, %Y, %r')
+    # do something horrifying 
+    df['string-ts'].iloc[1] = "None"
```
Comment (ian-r-rose) [inline dags/public-health/covid19/shelter-to-esri.py]: Wow, this really worked?

Comment (hunterowens) [reply dags/public-health/covid19/shelter-to-esri.py]: unfortunately, yes?

Comment (hunterowens) [reply dags/public-health/covid19/shelter-to-esri.py]: image

Comment (ian-r-rose) [reply dags/public-health/covid19/shelter-to-esri.py]: Debugging AGOL is like navigating by brownian motion.

**Comment by hunterowens** [comment]
@ian-r-rose can you review?

**Pull request merged**